healthcheck/README.md

```diff
@@ -0,0 +1,55 @@
+Health checks
+=============
+
+#### Add a health check to a service without a HTTP server
+
+Create a health check server to check neo4j and the filter API every 30 seconds:
+```
+neoHealthCheck := neo4j.NewHealthCheckClient(neo4jConnPool)
+filterAPIHealthChecker := filterAPIClient.New(config.FilterAPIURL)
+
+healthChecker := healthcheck.NewServer(config.BindAddr, time.Second * 30, errorChannel,
```

Comment:
the time element was not clear to me, consider:
```
healthChecker := healthcheck.NewServer(
    config.BindAddr,
    config.HealthCheckInterval,
    errorChannel,
    filterAPIHealthChecker,
    neoHealthCheck,
)
```
(which happens to be my preferred formatting, too!)

Reply:
Done 👍